istioctl/pkg/writer/envoy/configdump/testdata/ecds/output.txt

```diff
@@ -0,0 +1,3 @@
+NAME
```

Comment:
can we include the type? Possibly shortened for well known types, at least stripping `type.googleapis.com`.

We are going to use ECDS for more than wasm